Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[workspace]
+members = [
+	"rust/ccommon-backend",
+	"rust/ccommon-derive",
+	"rust/ccommon-rs",
+    "rust/ccommon-sys",
+]
+
+[profile.dev]
+opt-level = 0
+debug = true
+rpath = false
+lto = false
+debug-assertions = true
+codegen-units = 1
+
+[profile.bench]
```

Comment:
I always keep the bench and release profiles the same. They are defined separately because of how cargo selects the profile to use. `cargo build` uses the debug profile. `cargo build --release` uses the release profile, which we always want to have optimized at the cost of increased build time. We currently aren't using the `cargo bench` functionality - but I like to have benchmarks match what release performance would be. In the future, we might want to use the `cargo bench` functionality for microbenchmarks, so I figured it was best to just include that here now.

These are cargo-culted (pun) from rpc-perf and rezolus manifests. They only have an effect on binaries generated within this workspace, projects outside of this workspace which depend on library crates within ccommon would continue to use their individual (or workspace) profiles.

Even though we're unlikely to produce "real" binaries within this workspace, I would leave the bench and release profiles defined here as reference, accuracy of future benchmarks, and testing compilation of any example binaries we might include in the future.

Reply:
I've reduced the profile configs to the minimum to represent our overrides for `release` and `bench`